sdk-springboot/sdk-spring-core/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr.spring</groupId>
+    <artifactId>dapr-sdk-spring-parent</artifactId>
+    <version>1.12.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>dapr-sdk-spring-core</artifactId>
+  <name>Dapr Spring Core</name>
+  <description>Dapr Spring Core</description>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-keyvalue</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.diagrid.dapr</groupId>
+      <artifactId>testcontainers-dapr</artifactId>
+      <version>${testcontainers-dapr.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>${testcontainers-test.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${testcontainers-test.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>mysql</artifactId>
+      <version>${testcontainers-test.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
+    <dependency>
+      <groupId>com.mysql</groupId>
+      <artifactId>mysql-connector-j</artifactId>
```

Comment:
@artur-ciocanu why do we need this? We are not supposed to be accessing MySQL directly, even for tests.

Reply:
Great question! So I had a chat with Testcontainers folks on their Slack and I have found the following:
- PostgreSQL and MySQL both derive from `JdbcDatabaseContainer`, so potentially to use any of these containers you need a JDBC driver.
- A container can provide its own custom so called `WaitStrategy` and MySQL uses the default one based on a SQL query, while PostgreSQL uses one based on a simple log message.

Hence for MySQL we need a JDBC driver, if it is critical, we can implement our own MySQL container and provide a custom wait strategy similar to the one used by PostgreSQL.

Reply:
@artur-ciocanu I think all we need is a comment in the XML why the dependency is there.. so we know it is on purpose